File: sdks/csharp/src/Gemina.Sdk/Helpers/SdkVersion.cs
namespace Gemina.Sdk
{
    /// <summary>
    /// Version of the hand-written Gemina SDK package. This is the single
    /// source of truth consumed by the user-agent string; the generated
    /// client's metadata version is irrelevant.
    /// </summary>
    public static class SdkVersion
    {
        /// <summary>The package version, kept in sync with the csproj.</summary>
        public const string Version = "0.1.1";
    }
}

File: sdks/java/src/main/java/co/gemina/sdk/SdkVersion.java
package co.gemina.sdk;

/**
 * The SDK package version. This is the single hand-written source of truth for
 * the version consumed by the {@code User-Agent} string
 * ({@code gemina-sdk-java/<version>}); the generated client's own metadata
 * version is irrelevant and discarded.
 */
public final class SdkVersion {

    /** The gemina-sdk package version. Keep in sync with {@code pom.xml}. */
    public static final String VERSION = "0.2.1";

    private SdkVersion() {
    }
}
